src/test/java/net/datafaker/providers/base/ZodiacTest.java

```diff
@@ -0,0 +1,14 @@
+package net.datafaker.providers.base;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ZodiacTest extends BaseFakerTest<BaseFaker> {
+
+    @Test
+    void testSign() {
+        assertThat(faker.zodiac().sign()).isNotEmpty();
+    }
```

Comment:
I"m happy to align this with datafaker-net/datafaker#657 if it's accepted.

Reply:
If you want to further improve the test, that is fine, but this is quite in line with a majority of the test (most of them have been generated, and just make sure the basics work), so happy to accept it as is.